.vib/flink/goss/vars.yaml

```diff
@@ -0,0 +1,23 @@
+binaries:
+  - flink
+  - java
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/flink
+      - /opt/bitnami/flink
+      - /opt/bitnami/flink/conf
+      - /opt/bitnami/flink/bin
+root_dir: /opt/bitnami
+version:
+  bin_name: flink
+  flag: -v
+user:
+  flink:
+    exists: true
+group:
+  flink:
+    exists: true
```

Comment:
User and group tests are not rendered in the `vars.yaml`.

Instead, you should move them under `flink.yaml`

Reply:
You are right! Good catch!